admin/resources/js/stories/Input.stories.tsx

```diff
@@ -27,6 +29,7 @@ TextInput.args = {
   label: "First Name",
   name: "firstName",
   rules: { required: "This field is required", maxLength: 100 },
```

Comment:
I don't see `maxLength` value having an effect on submit or when typing, is it supposed to show an error on submit?

Reply:
Oh it looks like I forgot to add a message to the maxLength rule.